lib/index.js

```diff
@@ -74,12 +75,30 @@ class BEMClassName {
     }
 
     /**
+     * Adds optional string to the chain.
+     * Will be skiped if the argument equals null.
+     *
+     * @param {?String} value
+     * @return {BEMClassName}
+     */
+    concat(value) {
+        if (typeof value !== 'string' && !value) {
```

Comment:
Seems it should be `||`? Skip the step if the value is not a string **or** empty string?